config/management/src/storage.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::Error;
+use libra_crypto::{ed25519::Ed25519PublicKey, x25519};
+use libra_secure_storage::{CryptoStorage, KVStorage, Storage, Value};
+
+pub fn get_value(
```

Comment:
why the gets?
https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
I was trying to figure out a better way to collapse all the error handling.  I guess Java is stuck in my head for the `get`.